packages/sn-filter-pane/src/components/ListboxGrid/ListboxGrid.tsx

```diff
@@ -80,6 +83,8 @@ function ListboxGrid() {
                         ? <ListboxContainer
                           layout={item.layout}
                           borderBottom={(column.items?.length === j + 1) || !item.fullyExpanded}
+                          disableSearch={item.cardinal <= 3}
```

Comment:
Why: `<= 3`?

Reply:
It starts to look weird at that point, could even have been a larger number. Maybe 5 or 7. But there will be a specific property to disable search later on.